tick/inference/inference.py

```diff
@@ -0,0 +1,23 @@
+# License: BSD 3 clause
```

Comment:
This file is a duplication of tick/inference/tests/inference.py and should be deleted